hw/xbox/nv2a_vsh.h

```diff
@@ -24,6 +24,25 @@
 
 #include "qapi/qmp/qstring.h"
 
+enum Texgen {
```

Comment:
Yes.
I just called it that because there was also "enum AlphaFunc".
That should probably be renamed to PshAlphaFunc then, too?

Thanks for the feedback!